dev/dev.clj

```diff
@@ -106,8 +134,9 @@
                                   :kafka-config :kafka-config}}}})
 
 
-;; swap for `embedded-kafka-config` to use embedded-kafka
-(ir/set-prep! (fn [] standalone-config))
+;; swap for `embedded-kafka-config`  to use embedded-kafka
+(ir/set-prep! (fn [] checkpoint-fs-config))
```

Comment:
fine that you've popped an example config in here, thanks - but could we preserve the `standalone-config` as the default?